webpack/globalIdHelper.js

```diff
@@ -0,0 +1,13 @@
+const idSeparator = '-';
+const versionSeparator = ':';
+const defaultVersion = '01';
+
+export const decodeModelId = model => decodeId(model.id);
+
+export const decodeId = id => {
+  const split = atob(id).split(idSeparator);
+  return parseInt(split[split.length - 1], 10);
+};
+
+export const encodeId = (typename, id) =>
+  btoa([defaultVersion, versionSeparator, typename, idSeparator, id].join(''));
```

Comment:
This is imported from foreman_ansible. Would be nice if some shared package provided this

Reply:
This can be added to `foreman/webpack/assets/javascripts/react_app/common/helpers.js` if it's relevant

Reply:
https://projects.theforeman.org/issues/34614